devicesetup/src/main/java/io/particle/android/sdk/devicesetup/ApConnector.java

```diff
@@ -147,6 +146,14 @@ public SSID connectToAP(final WifiConfiguration config, Client client) {
             setupRunnables.add(() -> {
                 log.d("Adding network " + configSSID);
                 networkID.set(wifiFacade.addNetwork(config));
+
+                if (networkID.get() == -1) {
```

Comment:
It's "addNetwork() added the network, but reported failure (-1)", removing network did not report failure.